htdocs/xoops_lib/Xoops/Core/Logger.php

```diff
@@ -204,7 +204,7 @@ private function reportFatalError($msg)
         if ('cli' === php_sapi_name()) {
             fprintf(STDERR, "\nError : %s\n", $msg);
         } else {
-            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, $msg);
+            printf(_XOOPS_FATAL_MESSAGE, XOOPS_URL, utf8_encode($msg));
```

Comment:
Yes , \Patchwork\Utf8::filter($msg); works fine!
The error message was: "An exception occured in driver: SQLSTATE[HY000] [2002] Nenhuma ligação pôde ser feita porque o computador de destino as recusou ativamente.".

It came from an PDO Exception that simple extends php runtime exception. A file search on XoopsCore did not return any result for the {message}. I've checked my wamp installation and I did not find any configuration to change the locale. I think this message comes from the OS.